packages/mobile/src/invite/saga.ts

```diff
@@ -346,26 +344,10 @@ export function* skipInvite() {
 function* addTempAccountToWallet(inviteCode: string) {
   Logger.debug(TAG + '@addTempAccountToWallet', 'Attempting to add temp wallet')
   try {
-    const contractKit = yield call(getContractKit)
-    let tempAccount: string | null = null
-    const fornoMode = yield select(fornoSelector)
-    if (fornoMode) {
-      tempAccount = contractKit.web3.eth.accounts.privateKeyToAccount(inviteCode).address
-      Logger.debug(
-        TAG + '@redeemInviteCode',
-        'web3 is connected:',
-        String(yield call(contractKit.web3.eth.net.isListening))
-      )
-      addLocalAccount(inviteCode)
-    } else {
-      // Import account into the local geth node
-      tempAccount = yield call(
-        contractKit.web3.eth.personal.importRawKey,
-        trimLeading0x(inviteCode),
-        TEMP_PW
-      )
-    }
-    Logger.debug(TAG + '@addTempAccountToWallet', 'Account added', tempAccount!)
+    // Import account into the local geth node
+    const wallet = yield call(getWallet)
+    const tempAccount = yield call(wallet.addAccount, inviteCode, TEMP_PW)
```

Comment:
need to have `yield call([wallet, wallet.addAccount] , inviteCode, TEMP_PW)` otherwise the context isn't set. i can tell you didn't run this code 😄

Reply:
I don't understand why the tests are passing?... I mocked these functions but maybe incorrectly